graylog2-server/src/main/java/org/graylog/plugins/sidecar/rest/models/NodeDetails.java

```diff
@@ -50,12 +50,17 @@ public abstract class NodeDetails {
     @Nullable
     public abstract CollectorStatusList statusList();
 
+    @JsonProperty("tags")
+    @Nullable
+    public abstract List<String> tags();
```

Comment:
Shouldn't this be a Set to prevent duplicates?

Reply:
wouldn't hurt yeah. can I fix this in a separate PR? I'm waiting for this one to be merged